runner/README.md

```diff
@@ -0,0 +1,41 @@
+package runner is structs and interfaces to run Scoot processes.
+
+## Summary ##
+* Invoker runs one Command using Execer (and snapshot.Filer, etc.)
+* Controller controls when to Run something (using an Invoker)
+* Statuses queries Process history
+  * StatusesSync doesn't offer waiting, so requires client polling
```

Comment:
we should describe features in terms of what they do and only secondary in terms of what they don't do....
StatusesSync - returns the current status of requested runs.  This feature is exposed to the user (via entry points in the controller?)
LegacyStatuses - (is this really the name for the entry point?) - what does it do (what did the old Read API do?)
StatusWriter - creates and updates status entries. (be careful of the word 'Process' - not sure its well-defined at this point).  This feature is used by the Controller.

what about erasing statuses?
also - we are no longer offering the users a (blocking) 'wait for state' feature?  (I'm fine with that, just want to make sure)